conn.go

```diff
@@ -87,6 +123,7 @@ func (c *conn) Break() error {
 		return nil
 	}
 	if C.dpiConn_breakExecution(c.dpiConn) == C.DPI_FAILURE {
+		Log("msg", "Break", "error", c.getError())
```

Comment:
Log may be nil, so please guard it with "if Log != nil"

Reply:
Thanks . done